components/heading-4/bem.stories.mdx

```diff
@@ -0,0 +1,60 @@
+<!-- @license CC0-1.0 -->
+
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { ComponentTokensSection } from "../../documentation/components/ComponentTokensSection";
+import tokens from "../../proprietary/design-tokens/dist/index.json";
+import "../heading-4/bem.scss";
```

Comment:
I kind of expected the import order ESLint plugin to complain that the `.css` import (side effect imports) should be the last.

Reply:
The import order in mdx files isn't checked by our linting. But I've moved those side-effect imports to be the last import of the heading mdx files